public/apps/configuration/panels/role-edit/cluster-permission-panel.tsx

```diff
@@ -0,0 +1,61 @@
+
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import { ComboBoxOptions } from './types';
+import React, { Dispatch, SetStateAction } from 'react';
+import { PanelWithHeader } from '../../utils/panel-with-header';
+import { EuiForm, EuiFlexGroup, EuiFlexItem, EuiComboBox, EuiButton } from '@elastic/eui';
+import { FormRow } from '../../utils/form-row';
+
+export function ClusterPermissionPanel(props: {
+  state: ComboBoxOptions;
+  optionUniverse: ComboBoxOptions;
```

Comment:
Just curious how the name optionUniverse is created. Is it a terminology in EUI?

Reply:
Nope, I took the word from math set theory to indicate it is a set of all options rather to differ from selected options.